lib/libzfs/libzfs_util.c

```diff
@@ -241,12 +241,25 @@ libzfs_error_description(libzfs_handle_t *hdl)
 		    "into a new one"));
 	case EZFS_SCRUB_PAUSED:
 		return (dgettext(TEXT_DOMAIN, "scrub is paused; "
-		    "use 'zpool scrub' to resume"));
+		    "use 'zpool scrub' to resume scrub"));
+	case EZFS_SCRUB_PAUSED_TO_CANCEL:
+		return (dgettext(TEXT_DOMAIN, "scrub is paused; "
+		    "use 'zpool scrub' to resume or 'zpool scrub -s' to "
+		    "cancel scrub"));
 	case EZFS_SCRUBBING:
 		return (dgettext(TEXT_DOMAIN, "currently scrubbing; "
-		    "use 'zpool scrub -s' to cancel current scrub"));
+		    "use 'zpool scrub -s' to cancel scrub"));
+	case EZFS_ERRORSCRUBBING:
+		return (dgettext(TEXT_DOMAIN, "currently error scrubbing; "
+		    "use 'zpool scrub -e -s' to cancel error scrub"));
+	case EZFS_ERRORSCRUB_PAUSED:
+		return (dgettext(TEXT_DOMAIN, "error scrub is paused; "
+		    "use 'zpool scrub -e' to resume error scrub"));
```

Comment:
It seems to me that after launching an error scrub, you should be able to pause it with `zpool scrub -p` and stop it with `zpool scrub -s` just as you would a normal scrub.  Am I missing some reason why you would need the `-e` there as well?

Reply:
The original semantic of this PR was that the error scrub is completely independent of scrub. I didn't modify that and the "-e" switch is required to control an error scrub. However the two (scrub and error scrub) are mutually exclusive, ie only one of them may be active or paused at any given moment.

Reply:
I modified the behaviour so that an error scrub can be paused or canceled alone with "-p" or "-s" without needing the "-e" switch.